python_src/tests/aws/test_s3_utils.py

```diff
@@ -14,18 +14,18 @@
 from botocore.stub import Stubber
 from botocore.stub import ANY
 
-from lib.s3_utils import file_list_from_s3
-from lib.s3_utils import move_s3_objects
+from lamp_py.aws.s3 import file_list_from_s3_ingestion
+from lamp_py.aws.s3 import move_s3_objects
 
-TEST_FILE_DIR = os.path.join(os.path.dirname(__file__), "test_files")
+from ..test_resources import incoming_dir
```

Comment:
use this import to get the test files dir. should probably be a fixture, but this was easier to get off the ground.